tasks.py

```diff
@@ -43,21 +43,26 @@ def imageid(ctx):
         format='{{.Id}}'
     ))
 
+
 def link_phutil_map(ctx):
-    ctx.run("ln -fs `pwd`/phutil_map /tmp/phutil_map")
+    """Create link to phutil_map"""
+    ctx.run("cp `pwd`/phutil_map /tmp/phutil_map")
+
```

Comment:
The linked `phutil_map` isn't needed for test. It is required only to update when using liberate.
This change would be OK, if after `invoke liberate` the `phutil_map` would be copied back to `pwd`.
We could also consider an additional `docker-compose.dev.yml` file.